articles/connections/database/custom-db/templates/get-user.md

```diff
@@ -56,141 +56,240 @@ Auth0 provides sample scripts for use with the following languages/technologies:
+    server: 'the server',
+    options: {
+      database: 'the db name',
+      encrypt: true // for Windows Azure
+    }
+  });
 
-  callback(null, profile);
+  connection.on('debug', function(text) {
+    // if you have connection issues, uncomment this to get more detailed info
+    //console.log(text);
+  }).on('errorMessage', function(text) {
+    // this will show any errors when connecting to the SQL database or with the SQL statements
+    console.log(JSON.stringify(text));
+  });
+
+  connection.on('connect', function(err) {
+    if (err) return callback(err);
+
+    var user = {};
+    const query =
+      'SELECT webpages_Membership.UserId, UserName, UserProfile.UserName from webpages_Membership ' +
+      'INNER JOIN UserProfile ON UserProfile.UserId = webpages_Membership.UserId ' +
+      'WHERE UserProfile.UserName = @Username';
+
+    const getMembershipQuery = new Request(query, function (err, rowCount) {
+      if (err) return callback(err);
+      if (rowCount < 1) return callback();
+
+      callback(null, user);
+    });
+
+    getMembershipQuery.addParameter('Username', TYPES.VarChar, email);
+
+    getMembershipQuery.on('row', function (fields) {
+      user = {
+        user_id: fields.UserId.value,
+        nickname: fields.UserName.value,
+        email: fields.UserName.value
+      };
+    });
+
+    connection.execSql(getMembershipQuery);
+  });
 }
 ```
 
 ### MongoDB
 
 ```
-function getByEmail (name, callback) {
-  var profile = {
-    user_id:     "103547991597142817347",
-    nickname:    "johnfoo",
-    email:       "johnfoo@gmail.com",
-    name:        "John Foo",
-    given_name:  "John",
-    family_name: "Foo"
-  };
+function getByEmail(email, callback) {
+  const MongoClient = require('mongodb@3.1.4').MongoClient;
+  const client = new MongoClient('mongodb://user:pass@mymongoserver.com');
 
-  callback(null, profile);
+  client.connect(function (err) {
+    if (err) return callback(err);
+
+    const db = client.db('db-name');
+    const users = db.collection('users');
+
+    users.findOne({ email: email }, function (err, user) {
+      client.close();
+
+      if (err) return callback(err);
+      if (!user) return callback(null, null);
+
+      return callback(null, {
+        user_id: user._id.toString(),
+        nickname: user.nickname,
+        email: user.email
+      });
+    });
+  });
 }
 ```
 
 ### MySQL
 
 ```
-function getByEmail (name, callback) {
-  var profile = {
-    user_id:     "103547991597142817347",
-    nickname:    "johnfoo",
-    email:       "johnfoo@gmail.com",
-    name:        "John Foo",
-    given_name:  "John",
-    family_name: "Foo"
-  };
+function getByEmail(email, callback) {
+  const mysql = require('mysql');
 
-  callback(null, profile);
+  const connection = mysql({
+    host: 'localhost',
+    user: 'me',
+    password: 'secret',
+    database: 'mydb'
+  });
+
+  connection.connect();
+
+  const query = 'SELECT id, nickname, email FROM users WHERE email = ?';
```

Comment:
In Heroku review app, code is not rendered correctly from here on (green, when should not be):
https://docs-content-staging-pr-8876.herokuapp.com/docs/connections/database/custom-db/templates/login#mysql